VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/quick-links/quick-links.component.ts

```diff
@@ -17,10 +17,16 @@ import { pageUrls } from 'src/app/shared/page-url.constants';
 export class QuickLinksComponent implements OnInit, OnDestroy {
     private loggerPrefix = '[QuickLinksComponent] -';
 
+    @ViewChild('fullName', { static: false })
+    inputFullName: ElementRef;
```

Comment:
Why is this on 2 separate lines? Does prettier do this?

Reply:
It looks like some others in the project are like this and some aren't. Lets keep it on 1 line if possible to make it obvious they are related.

Reply:
that is done now